### PR TITLE
Container-preloading improvements and minor fixes

### PR DESCRIPTION
**Pull request by caiotpereira**
Typo: aktualizr-lite.rst
Code format: linux-distro.rst
Improvements in container-preloading.rst
 - Define only 'preload' to avoid mixing pre-load/preload
 - Add emphasis to assembly-system-image in preloaded containers.

### CODE REVIEW

**Comment by doanac** [comment]
Docs for fab90ffd209cdcc4bbea2f16d9c5e0f93f4c82cb are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1360/docs/artifacts/html/index.html

**Review by kprosise — APPROVED** [review]
Looks good to me!

File: source/user-guide/container-preloading/container-preloading.rst [hunk 1]
```diff
@@ -110,7 +110,7 @@ The latest **Target** named ``containers-devel`` should be the CI job you just c
 
    FoundriesFactory New Target
 
-When FoundriesFactory CI finishes all jobs, click in the **Target**, find :guilabel:`Runs` and download the image:
+When FoundriesFactory CI finishes all jobs, click in the **Target**, find :guilabel:`Runs` , **assembly-system-image**, **devel** folder and download the image:
```
Comment (mike-sul) [inline source/user-guide/container-preloading/container-preloading.rst:113]: The folder name (`devel` in this case) depends on a branch/ref/tag the run is triggered for.

Comment (mike-sul) [reply source/user-guide/container-preloading/container-preloading.rst:113]: Although, in this specific image `container-preloading-image.png` it's indeed `devel` folder. So, maybe just add a comment that this folder name corresponds to a branch/ref name.

**Review by mike-sul — APPROVED** [review]

File: source/user-guide/container-preloading/container-preloading.rst (hunk 1 above)
Comment (caiotpereira) [reply source/user-guide/container-preloading/container-preloading.rst:113]: @mike-sul  good catch, I will update this information.
There's one thing that I was wondering this morning , to add a note to regarding the u-boot files /mfg tools, they  still in the same folder as usual.

**Comment by doanac** [comment]
Docs for a601f66021c942102928c226f437f850f4f2512c are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1361/docs/artifacts/html/index.html

**Review by mike-sul — APPROVED** [review]

**Pull request merged**